matchbook-sdk-rest/src/main/java/com/matchbook/sdk/rest/HttpConfig.java

```diff
@@ -35,21 +37,21 @@ public String toString() {
 
     public static class Builder {
 
-        private int connectionTimeout = 15_000;
-        private int writeTimeout = 15_000;
-        private int readTimeout = 15_000;
+        private long connectionTimeout = TimeUnit.SECONDS.toMillis(5);
```

Comment:
why long not int?

Reply:
`OkHttp` library expects a long value and also other HTTP libs I now (e.g. Apache) expect long as well. I don't see why we should limit to int and then convert those values to long under the hoods.